### PR TITLE
[ADD] dead_mans_switch_client

### PR DESCRIPTION
**Pull request by hbrunn**
see https://github.com/OCA/server-tools/pull/297 for discussion

### CODE REVIEW

File: dead_mans_switch_client/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+After installing this module, you need to fill in the system parameter
+`dead_mans_switch_client.url`. This must be the full URL to the server's
+controller, usually of the form https://your.server/dead_mans_switch/alive
+
+This module attempts to send CPU and RAM statistics to the server. While this
+is not mandatory, it's helpful for assessing a server's health. If you want
+this, you need to install `psutil`.
+
+You can also have the currently online users logged, but this only works if
```
Comment (elicoidal) [inline dead_mans_switch_client/__openerp__.py]: The module is removed AFAIK

File: dead_mans_switch_client/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+=============
+
+After installing this module, you need to fill in the system parameter
+`dead_mans_switch_client.url`. This must be the full URL to the server's
+controller, usually of the form https://your.server/dead_mans_switch/alive
+
+This module attempts to send CPU and RAM statistics to the server. While this
+is not mandatory, it's helpful for assessing a server's health. If you want
+this, you need to install `psutil`.
+
```
Comment (elicoidal) [inline dead_mans_switch_client/__openerp__.py]: Maybe add the same kind of warning than in v8 branch

**Comment by elicoidal** [comment]
LGTM (no test yet)

File: dead_mans_switch_client/__openerp__.py (hunk 2 above)
Comment (hbrunn) [inline dead_mans_switch_client/__openerp__.py]: which one do you mean? This is a verbatim copy of 8.0's readme

File: dead_mans_switch_client/__openerp__.py (hunk 1 above)
Comment (hbrunn) [inline dead_mans_switch_client/__openerp__.py]: true, that should go away

File: dead_mans_switch_client/__openerp__.py (hunk 2 above)
Comment (elicoidal) [inline dead_mans_switch_client/__openerp__.py]: @hbrunn 
About this: https://github.com/hbrunn/server-tools/commit/7be1aa3059fb7c96fdf574013228e1e637bb98cb
(or maybe you consider it only at server level?)

Comment (hbrunn) [inline dead_mans_switch_client/__openerp__.py]: ah, indeed. Not relevant for the client in my opinion

**Comment by StefanRijnhart** [comment]
Pending this one: https://github.com/hbrunn/server-tools/pull/1/files

**Comment by elicoidal** [comment]
This one I will take care and add his github to Elico Corp organization

**Comment by elicoidal** [comment]
CLA is OK

**Comment by gurneyalex** [comment]
@hbrunn there seem to be an error in the travis tests directly related to this module. Can you check it out? https://travis-ci.org/OCA/server-tools/jobs/97191775#L334

**Comment by hbrunn** [comment]
@gurneyalex thanks for the heads up, should be fixed now

**Comment by hbrunn** [comment]
@elicoidal @StefanRijnhart @gurneyalex thanks for your points, now with a green travis :-)

**Comment by gurneyalex** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1: I think we can merge this one

**Pull request merged**